### PR TITLE
Handles SDK package rename; file -> files

### PR DESCRIPTION
**Pull request by jtreminio-dropbox**
Replaces #14

### CODE REVIEW

**Comment by jtreminio-dropbox** [comment]
Failures are expected:

> Response : test_signature_request_create_embedded {'error': {'error_msg': 'Must specify file(s) to be sent.', 'error_path': 'file', 'error_name': 'bad_request'}}

**Comment by fenghao-chen** [comment]
For C#, errors are `stderr=b"Unable to find image 'hellosign/dropbox-sign-dotnet:latest' locally\ndocker: Error response from daemon: pull access denied for hellosign/dropbox-sign-dotnet, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.\nSee 'docker run --help'.\n")`

For Java, errors are `stderr=b'Error: Could not find or load main class com.dropbox.sign.sdk.Requester\nCaused by: java.lang.ClassNotFoundException: com.dropbox.sign.sdk.Requester\n')`

For Python, those two account related tests failed due to `dropbox_sign.exceptions.ApiTypeError: Invalid type for variable \'account_create_request\'. Required value type is AccountCreateRequest and passed type was AccountCreateResponse at [\'account_create_request\']\n')`

The detailed error messages are hard to spot. You will need to scroll up the log quite a bit

File: data_java/Requester.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
-package com.hellosign.sdk;
+package com.dropbox.sign_sandbox;
```
Comment (fenghao-chen) [inline data_java/Requester.java:1]: this should be `com.dropbox.sign.sdk`, which matches the path in docker file above

File: data_python/requester.py [hunk 2]
```diff
@@ -163,7 +155,7 @@ def _account_api(self):
         api = apis.AccountApi(self._api_client)
 
         if self._operation_id == 'accountCreate':
-            obj = self._serialize(models.AccountCreateRequest)
+            obj = m.AccountCreateResponse.init(self._data)
```
Comment (fenghao-chen) [inline data_python/requester.py:158]: should be `m.AccountCreateRequest.init(self._data)`

**Review by fenghao-chen — CHANGES_REQUESTED** [review]

**Review by fenghao-chen — APPROVED** [review]
LGTM. Let's wait for the HelloFax release then merge this PR, so that we don't get false alarm in between

**Review by fenghao-chen — APPROVED** [review]
LGTM

**Pull request merged**